template/bash_profile.template.bash

```diff
@@ -31,6 +31,11 @@ export SCM_CHECK=true
 # Will otherwise fall back on $HOSTNAME.
 #export SHORT_HOSTNAME=$(hostname -s)
 
+# Set Xterm/screen/Tmux title with only a short username.
+# Uncomment this (or set SHORT_USER to something else),
+# Will otherwise fall back on $USER.
+#export SHORT_USER=$(echo $USER | sed -r 's/^(.{3}).{3,}(.{3})$/\1..\2/')
```

Comment:
How would you solve the problem with the sed expression in this case? The one you have will only work with Gnu sed...